libs/catalog/src/lib/site-title/site-title.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core'
+import { Observable } from 'rxjs'
+import { SettingsListResponseApiModel, SiteApiService } from '@lib/gn-api'
+import { shareReplay } from 'rxjs/operators'
+
+@Component({
+  selector: 'catalog-site-title',
+  templateUrl: './site-title.component.html',
+  styleUrls: ['./site-title.component.css'],
+})
+export class SiteTitleComponent implements OnInit {
+  siteInfo$: Observable<SettingsListResponseApiModel>
+
+  constructor(private siteApiService: SiteApiService) {}
+
+  ngOnInit(): void {
+    this.siteInfo$ = this.siteApiService.get4().pipe(shareReplay())
```

Comment:
I think this should be in a service, in your case, `siteInfo$` is a shared observable but only within the component.
If another component do the same thing, it will still perform the request again.